content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst

```diff
@@ -0,0 +1,32 @@
+============
+Cycle Counts
```

Comment:
- Delete `Counts`
- Replace with: `counts` b/c headers should be in sentence case